pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -116,7 +116,7 @@ func init() {
 				// BuildControllerFactory.buildDeleteLW
 				{
 					Verbs:     sets.NewString("get", "list", "create", "delete"),
-					Resources: sets.NewString("pods"),
```

Comment:
You don't need all actions to apply to the pods/bind resource ... create should be sufficient